platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -138,6 +139,8 @@
     }
 }
 
+# Global logger class instance
+logger = Logger()
```

Comment:
Loger is not needed. It is Python. Exception is enough

Reply:
Removed